Testscripts/Linux/XDP-Action.sh

```diff
@@ -60,7 +62,7 @@ do
 	if [ $? -eq 1 ]; then
 		LogErr "ERROR: Please provide valide client and server ip. Invalid ip: ${ip}"
 		SetTestStateAborted
-		exit 0
+		exit 1
```

Comment:
> I'm confused that what value we should exit with. @johnsongeorge-w had comments in my PR that here we should exit 0.

https://github.com/LIS/LISAv2/pull/987/files#diff-1c8e9b52afe93480ee95409095096678R81 this can handle exit code not equal 0, I just keep the same style in the bash script.